src/Sentry/Internal/Hub.cs

```diff
@@ -184,8 +193,19 @@ public SentryId CaptureEvent(SentryEvent evt, Scope? scope = null)
             {
                 var currentScope = ScopeManager.GetCurrent();
                 var actualScope = scope ?? currentScope.Key;
+
+                // Inject trace information from a linked span
+                if (evt.Exception is { } exception &&
```

Comment:
I notice d we're not taking trace info from the scopeif there's a span kn the scope we can connect events to it through the span bound to the scope (or currently the transaction property). I guess separate PR for that. This is what other sdks do btw so we should have this on the sdk docs as a spec otherwise no connection between errors and transactions